webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,44 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+
+    updateConversationMetadata: function (conversationMetadata) {
+        if (conversationMetadata) {
+            if (
+                conversationMetadata.groupId &&
+                conversationMetadata.metadata &&
+                kommunicateCommons.isObject(conversationMetadata) &&
```

Comment:
lets say if kommunicateCommons.isObject(conversationMetadata) returns false then will the code before it work fine? 
conversationMetadata.groupId &&
                conversationMetadata.metadata &&

Reply:
okay got it will update the order